src/NsisoLauncherCore/Auth/IAuthenticator.cs

```diff
@@ -26,11 +29,6 @@ public class AuthenticateResult
         public Uuid SelectedProfileUUID { get; set; }
 
         public UserData UserData { get; set; }
-
-        public AuthenticateResult(AuthState state)
-        {
-            this.State = state;
-        }
     }
 
     public enum AuthState
```

Comment:
According to the GitHub action's warning, you should remove the underscores from member name NsisoLauncherCore.Auth.AuthState.
I don't know why they suggest doing this, but I think it is better to fix it.